frontend/src/app/shared/components/global-footer/global-footer.component.scss

```diff
@@ -63,7 +63,6 @@ footer .row.main .links .category:not(:first-child) {
 }
 
 footer .site-options {
-  float: right;
```

Comment:
@nymkappa @softsimon I don't know where this needs to be raised, but please use the `float-right` or `float-left` CSS classes on your elements, instead of manually setting it like this. Because if we try to abide by this simple rule, then everything would :sparkles: automagically :sparkles:  be correct in the RTL mode. (thanks to overrides that `.rtl-layout` is doing [here](https://github.com/mempool/mempool/blob/4169e1053ffbdf1ec9fcbe00016db257766fcd5d/frontend/src/styles.scss#L862-L876))